test.lua

```diff
@@ -343,7 +343,14 @@ test:test('simple expires test',  function(test)
     test:is(task.name, "test", 'checking task name')
     local restarts = 1
     test:is(task.restarts, restarts, 'checking restart count')
```

Comment:
Shouldn't restart be checked in the same way?

Reply:
I believe no. `task.restarts` is a simple counter that changes synchronously, while the `expired_tuples_count` is a counter too, but it depends on the asynchronous operation.